modules/tinymce/src/core/main/ts/api/commands/NewlineCommands.ts

```diff
@@ -1,19 +1,20 @@
+import * as InsertBlock from '../../newline/InsertBlock';
 import * as InsertBr from '../../newline/InsertBr';
 import * as InsertNewLine from '../../newline/InsertNewLine';
 import Editor from '../Editor';
 
 export const registerCommands = (editor: Editor): void => {
   editor.editorCommands.addCommands({
     insertParagraph: () => {
-      InsertNewLine.insert(editor);
+      InsertNewLine.insertBreak(InsertBlock, editor);
```

Comment:
Using a module as an object has caused various issues before, so I really don't like seeing us do that here. For example, Alloy used to do this and then was using `Obj` with it and that worked until Rollup fixed a bug where it wasn't following the standard (basically modules shouldn't have a prototype) and this caused things to die.

Can we come up with another way of passing the insert function and fake event name?

Reply:
does `export const {}` work?

Reply:
yeah that should work fine 👍